### PR TITLE
249 accordions

### PR DESCRIPTION
**Pull request by emilyville**
on the patient form:
- Add some more radios where radios belong
- remove accordions (SO MUCH CLEANER YAY)
- just one submit button
- merge in asia.
- some css I'm not super proud of, but it'll do for now.

### CODE REVIEW

File: app/views/patients/_form.html.slim [hunk 1]
```diff
@@ -1,142 +1,109 @@
...
-            table
-              - for episode in @patient.episode_of_cares do
-                tr class="clickable-row" data-href=edit_polymorphic_url([@patient, episode.specific])
-                  td
-                    strong
-                    = episode.episode_date
-                  td
-                    = episode.specific.class.name
+  .form-inputs
+    h4 Personal Information
```
Comment (awong-dev) [inline app/views/patients/_form.html.slim]: Does this need .callout?

File: app/views/patients/_form.html.slim [hunk 2]
```diff
@@ -1,142 +1,109 @@
...
+      :item_wrapper_class => 'radio-inline'            
+    = f.input :sci_arrival_date, as: :string, input_html: { class: "datepicker" }
+    = f.input :is_on_active_duty, label: "Currently on Active Duty"
+    = f.input :sci_type, collection: Patient.sci_types.keys, :prompt => "Select one"
+    = f.input :date_of_injury, as: :string, input_html: { class: "datepicker" }
+
+    = f.simple_fields_for :asia_assessment do |asia_assessment|
+      = render 'shared/asia_fields', :f => asia_assessment    
+    
+    h4 Psychosocial
```
Comment (awong-dev) [inline app/views/patients/_form.html.slim]: Does this need .callout?

**Comment by awong-dev** [comment]
Looks good to me!

Flagged 2 possible errors. Merging first. Feel free to fix in separate PR if appropriate.

**Comment by emilyville** [comment]
The callouts are being used a bit randomly and the only change they make is visual. still sussing out a consistent way to use them - but the above was intentional!

**Pull request merged**